### PR TITLE
Use '--lb-logfile' command line option

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: eq/init.cpp [hunk 1]
```diff
@@ -295,6 +252,15 @@ bool _parseArguments( const int argc, char** argv )
         Global::setWorkDir( path.parent_path().string( ));
     }
 
+    for( int i = 1; i < argc; ++i )
+    {
+        if( std::string( argv[i] ) == "--eq-logfile" )
+        {
+            argv[i][2] = 'l'; // rewrite to --lb-logfile
```
Comment (tribal-tec) [inline eq/init.cpp]: nice one :)

**Comment by tribal-tec** [comment]
+1

**Comment by eile** [comment]
retest this please.

**Comment by eile** [comment]
retest this please.

**Pull request merged**